### DISCUSSION TITLE
Managing the team configuration file

### DISCUSSION THREAD

**Discussion by jellypuno** [zowe/zowe-explorer-vscode#1535] [Squad Discussion]
As a Zowe Explorer user that leverages the team config file, I would like to have the ability to manage my team config via Zowe Explorer. 

Open Questions:
1. How are we going to provide the user the ability to manage the team config file?
2. Should we use a GUI?
3. Can we use YAML as an intermediate solution?

So far the recommendation that we have is to open the JSON file and the user will edit that file and they manually refresh VSCode after the update.

**Comment by jellypuno** [comment]
Goal: 
- Have a solution or and architecture that we can implement in the future.
- Answer the open questions above
- Roadmap for migration? We should be able to create an architecture that supports both team config and old profile folders
   - Zowe V3 is projected to be GA maybe in Feb 2024 (As per Mike B.'s comment below) 
   - Raise to CLI team regarding the usage of YAML files. That JSON files are not user friendly and YAML is better.

**Reply by jellypuno** [reply]
Microsoft publish a new WebView but we need to check with Che/Theia

**Reply by MikeBauerCA** [reply]
Zowe V3 is projected to be Active LTS in Feb 2024. You can view the release timeline at the bottom of the Zowe Download page -> https://www.zowe.org/download.html
